src/media/js/addon/selectors/addon.js

```diff
@@ -0,0 +1,13 @@
+export function addonListSelector(addons={}) {
+  return Object.keys(addons)
+               .sort()
+               .map(slug => addons[slug]);
+}
```

Comment:
Nit: I find this a lot more readable

    export function addonListSelector(addons={}) {
      return Object
        .keys(addons)
        .sort()
        .map(slug => addons[slug]);
    }